### DISCUSSION TITLE
Inventory tooltip flickering

### DISCUSSION THREAD

**Discussion by jouborges** [paternostrox/Minecraft4Unity#2] [Q&A] [answered]
When the inventory is opened and I hover the cursor over an item there is some flickering present in the tooltip, I suspect that this can be related to a raycasting/UI layering problem.

Also swapping items in the inventory works but I guess it should have an image following the mouse when the player picks an item, the picked item just disappears and I guess this is not intended.

**Comment by paternostrox** [comment, marked as answer]
It was indeed a raycasting issue, the tooltip itself was marked as a raycast target. Just pushed the correction to the repository, feel free to check it out.

Regarding the inventory item swapping I'm currently working on a fix right now. I'll also try to get the readme to a decent shape by the end of this week.
